.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,8 @@
+-   id: tmt-test-lint
+    name: tmt tests lint
+    entry: bash -c "git ls-files --error-unmatch .fmf/version && tmt tests lint --source $@" PAD
```

Comment:
From the man bash: `If the -c option is present, then commands are read from string. If there are arguments after the string, they are assigned to the positional parameters, starting with $0.`
So PAD is assigned to $0 and any changed file is added using $@.